### DISCUSSION TITLE
Portfolio em inglês ou português  ?

### DISCUSSION THREAD

**Discussion by ViniciusGularte** [frontendbr/forum#1201] [Discussão]
Fala pessoal, 

uma coisa que venho pensando a tempos, devo migrar todo meu portfólio  para linguá inglesa ? ou manter no pt-br ? Li alguns artigos que é mais bem visto um portfólio totalmente em inglês, outros que depende..

Alguns projetos e artigos que estou desenvolvendo, eu queria um alcance maior , porém outros são especificamente voltado pro pessoal br, não sei se ficaria legal, algumas coisas em inglês e outras em pt-br .. 

Como vocês organizam  seus portfólios e projetos ? Vocês definem um publico alvo ou definiram uma linguagem padrão para todos ?

Sobre meu foco no momento, eu moro no interior do Rio Grande do Sul, e  estou juntando dinheiro pra   me mudar pra uma cidade grande ( São Paulo, Porto Alegre...), então ainda não estou focando no mercado internacional(Mas sempre aberto a oportunidades :smile: ).

**Comment by leandrocunha** [comment]
O mercado fora do Brasil está bem aquecido, se sua intenção é atingir o mercado fora, faça em inglês. O ideal seria bilingue.

**Comment by felipefialho** [comment]
Como geralmente o ideal é não adicionar muitos textos no portfolio, acho uma boa ser em inglês. Meio que já tem potencial de atingir qualquer pessoa no mundo.

**Comment by phenriqueleao** [comment]
Como o @leandrocunha disse, acho interessante ser bilíngue, já que você deseja atingir os dois públicos. Um exemplo que acho muito legal é o do @rubenmarcus, [aqui](http://www.rubenmarcus.com/)

**Comment by diessica** [comment]
Tecnicamente, nada te impede de fazer os dois! Se ainda está começando sua carreira no Brasil, é legal focar no português também. Como é importante saber inglês de qualquer forma, você pode ir trabalhando em uma versão em inglês.

**Comment by nazarepiedady** [comment]
Faça um portifolio que possua um script de geolocalização e mude o idioma do seu portifolio segundo o local de acesso, se for um pais lusofono é português se não for é inglês.

**Reply by getulio-souza** [reply]
Como insiro esse script?

**Reply by nazarepiedady** [reply]
Você insere scripts usando o marcador de script, de que modo mais seria?

```js
<script src="example.js"></script>
```

**Reply by offreflix** [reply]
Acredito que ele quer saber como fazer um script de geolocalização

**Comment by victorkurauchi** [comment]
Acho interessante ser em inglês, até porque o conteúdo não vai ser muito extenso, e sim objetivo.
E acredito que isso vale pro seu perfil no linkedin também, é provável que "recrutadores" de outros países encontrem seu perfil (se esse for um interesse seu também).

Aliás, fiz o meu pelo http://ineedaresu.me/ haha

**Comment by ViniciusGularte** [comment]
Obrigado a todas colaborações, como sempre esse fórum está recheado de pessoas incríveis :heart: .

Segui algumas dicas e estou refatorando alguns projetos meus para ter o famoso botão PT/EN, e também ando escrevendo alguns artigos em inglês, @victorkurauchi não conhecia esse, achei bem legal e já recomendei pra uma galera que tava apanhando pra criar um bom currículo aqui na região.

**Reply by ferreiraitalo** [reply]
Olá, tudo bem?

"Segui algumas dicas e estou refatorando alguns projetos meus para ter o famoso botão PT/EN..." 

O que seria esse botão?

**Reply by ViniciusGularte** [reply]
Opa @ferreiraitalo, tudo bem, seria um switch pra mudar a linguagem do teu site, clicar e mudar pra portugues/ingles, na época eu achava interessante ... hoje em dia eu usaria alguma lib e pegaria automatico pelo região daonde o user está acessando, ou iria full inglês mesmo 😅

**Reply by ferreiraitalo** [reply]
Certo, muito obrigado meu amigo!!!

**Comment by tchiinhemba** [comment]
Deparei-me com essa dúvida agora mesmo... E aqui encontrei a resposta...
